tests/e2e/cypress/integration/eligibility.spec.js

```diff
@@ -0,0 +1,88 @@
+describe("Confirms a user’s eligibility status", () => {
+  beforeEach(() => {
+    cy.visit("/")
+    cy.contains("ABC")
+      .click()
+    cy.contains("Let’s do it!")
+      .click()
+  })
+
+  it("Takes user to eligibility confirmation page", () => {
+    cy.contains("Ready to continue")
+      .click()
+
+    cy.contains("Let’s see if we can confirm your age")
+    cy.contains("CA driver’s license or ID number *")
+  })
+
+  it("Confirmation page has a form with a driver’s license or ID number form label and corresponding form field", () => {
+    cy.contains("Ready to continue")
+      .click()
+    cy.get('input:focus').should('have.length', 0)
+
+    cy.contains("CA driver’s license or ID number *")
+      .click()
+
+    cy.get('input:focus').should('have.length', 1)
+  })
+
+  it("Confirmation page has a form with a lastname form label and corresponding form field", () => {
+    cy.contains("Ready to continue")
+      .click()
+    cy.get('input:focus').should('have.length', 0)
+
+    cy.contains("Last name (as it appears on ID) *")
+      .click()
+
+    cy.get('input:focus').should('have.length', 1)
+  })
```

Comment:
I tried to test that the form fields and labels are semantic (matching `for`/`name` values) in a user-centric way.

A user clicks on the label (`CA driver's license or ID number *`) and the form field should come into focus. For this, I'm testing that 1 form field comes into focus. Is that enough? (I could do an extra check to see that the form field and input are in the same parent div, or that the `name` ('sub') matches the `for` ('sub') -- but those seem like very code-centric things to check -- it's not checking for something a user would notice or be affected by. How else would a user know if a form label and field are correctly paired?)